### PR TITLE
Unify containers images used by CI and development environment 

### PR DESCRIPTION
**Pull request by eduardoj**
Taking advantage of the binary compatibility of both distributions, SLE 15 SP3 and openSUSE Leap 15.3, and after creating a new source of docker images based in SLE 15 SP3 (see https://build.opensuse.org/project/show/OBS:Server:Unstable:Containers ), we use those docker images for both, our CI setup and our development environment.

Upgrading the version of the distribution also upgraded the Chromium version. Two issues rised:
1. All feature tests failed: this was fixed by adding the `--disable-gpu` flag to the options of Chromium.
1. A false positive in feature package tests was fixed, making the tests for adding a user and adding a group fail. The bad
behavior was, that clicking the links 'Add User' or 'Add Group' was possible, even if the link was outside the visual area. ~Skipping~ Scrolling down to these links in these failing tests in mobile, make the tests pass again. See also [this comment](https://github.com/openSUSE/open-build-service/pull/9559#pullrequestreview-413770036), where it was considered, to scroll down until an element that should be clicked appeared into the visual area.

-----------------

If this PR requires any particular action or consideration before deployment,
please check the reasons or add your own to the list:

* [ ] Requires a database migration that can cause downtime. Postpone deployment until maintenance window[1].
* [ ] Contains a data migration that can cause temporary inconsistency, so should be run at a specific point of time.
* [ ] Changes some configuration files (e.g. options.yml), so the changes have to be applied manually in the reference server.
* [ ] A new Feature Toggle[2] should be enabled in the reference server.
* [ ] Proper documentation or announcement has to be published upfront since the introduced changes can confuse the users.

[1] https://github.com/openSUSE/open-build-service/wiki/Deployment-of-build.opensuse.org#when-there-are-migrations
[2] https://github.com/openSUSE/open-build-service/wiki/Feature-Toggles-%28Flipper%29#you-want-real-people-to-test-your-feature

### CODE REVIEW

File: src/api/spec/support/shared_examples/features/beta/user_tab.rb [hunk 1]
```diff
@@ -41,6 +41,8 @@
     end
 
     scenario 'Add non existent user' do
+      skip_on_mobile
```
Comment (danidoni) [inline src/api/spec/support/shared_examples/features/beta/user_tab.rb:44]: Can't we avoid skipping the test somehow?

Comment (eduardoj) [reply src/api/spec/support/shared_examples/features/beta/user_tab.rb:44]: As the description says, in [this comment](https://github.com/openSUSE/open-build-service/pull/9559#pullrequestreview-413770036) it was considered, to scroll down until an element that should be clicked appeared into the visual area.

I'm open to other alternatives, too.

**Comment by hennevogel** [comment]
There is also [obs-review-apps](https://github.com/openSUSE/obs-review-apps/blob/master/files/docker-compose.yml)

**Comment by eduardoj** [comment]
> There is also [obs-review-apps](https://github.com/openSUSE/obs-review-apps/blob/master/files/docker-compose.yml)

There you have it: https://github.com/openSUSE/obs-review-apps/pull/14

File: src/api/spec/support/shared_examples/features/beta/user_tab.rb (hunk 1 above)
Comment (hennevogel) [reply src/api/spec/support/shared_examples/features/beta/user_tab.rb:44]: doesn't has to happen here and now right?

**Review by hennevogel — APPROVED** [review]

File: src/api/spec/support/shared_examples/features/beta/user_tab.rb (hunk 1 above)
Comment (dmarcoux) [reply src/api/spec/support/shared_examples/features/beta/user_tab.rb]: I found a solution for this, scrolling to the element will work. See the commit I pushed.

**Review by dmarcoux — CHANGES_REQUESTED** [review]
Let's not skip the specs for mobile. I found a fix for this, the latest commit I pushed to the PR.

**Comment by eduardoj** [comment]
@dmarcoux:

> Let's not skip the specs for mobile. I found a fix for this, the latest commit I pushed to the PR.

Thanks! Applied. I modified the commit description accordingly. Please, have a look again.

File: src/api/spec/support/shared_examples/features/beta/user_tab.rb [hunk 2]
```diff
@@ -148,6 +154,8 @@
     end
 
     scenario 'Add non existent group' do
+      skip_on_mobile
```
Comment (dmarcoux) [inline src/api/spec/support/shared_examples/features/beta/user_tab.rb:157]: We can do the same here as in the examples with `Add User`, just switching `Add User` to `Add Group` and it should work.

Comment (eduardoj) [reply src/api/spec/support/shared_examples/features/beta/user_tab.rb:157]: On it.

Comment (eduardoj) [reply src/api/spec/support/shared_examples/features/beta/user_tab.rb]: Done. I fixed the commit description again. Please take a look.

Comment (dmarcoux) [reply src/api/spec/support/shared_examples/features/beta/user_tab.rb]: This part of the commit message isn't true anymore: `Or skip the test in mobile.` It should be removed.

Comment (dmarcoux) [reply src/api/spec/support/shared_examples/features/beta/user_tab.rb]: Done!

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**